test/state.cpp

```diff
@@ -82,12 +82,38 @@ TEST_CASE_FIXTURE(StateTest, "Two Person")
 
   // Handle the Add proposal and create a Commit
   auto add = first0.add_proposal(key_packages[1]);
-  auto [commit, welcome, first1] = first0.commit(fresh_secret(), { add });
+  auto opts = State::CommitOpts{ { add }, true };
```

Comment:
It would be nicer if we could put these back inline.  Maybe moving `CommitOpts` to the top level will make it compact enough?